backend_secure/web/ui/static/js/app.js

```diff
@@ -68,3 +67,123 @@
         processData: false,
     });
 }
+
+function add_new_todo(todo_status_id) {
+    let form_id = "#form_" + todo_status_id
+    $(form_id).trigger("reset");
+    var modal_id = "#modal_" + todo_status_id
+    $(form_id).removeClass('was-validated')
+    $(modal_id + " #submit_values").attr("onclick", "insert_new_todo(\"" + form_id + "\"," + "\"" + todo_status_id + "\")")
+    $(modal_id + " #add_todo_modal").attr("onclick", "close_modal_window(\"" + modal_id + "\")")
+    $(modal_id).modal('show')
+}
+
+function close_modal_window(modal_id){
+    $(modal_id).modal('hide');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/IrinaLokhvytska/owasp/pullRequest?prid=8118882)
```suggestion
    $(modal_id).modal("hide");
```